package.json

```diff
@@ -95,11 +95,11 @@
     "lazy-child": "^0.3.1",
     "polished": "^4.1.0",
     "prop-types": "^15.7.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-modal": "^3.11.1",
-    "react-oembed-container": "^1.0.0",
-    "react-swipeable": "^5.5.0",
+    "react-oembed-container": "^1.0.1",
```

Comment:
update here explicitly as there's a new update for react 17 support peer declaration